### PR TITLE
add some date unit tests

### PR DESCRIPTION
**Pull request by championswimmer**

### CODE REVIEW

File: app/src/test/java/org/fossasia/openevent/utils/DateTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import android.text.format.DateUtils;
+
+
+import org.junit.Test;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.Locale;
```
Comment (juslee) [inline app/src/test/java/org/fossasia/openevent/utils/DateTest.java]: Codacy Issue found: [Avoid unused imports such as 'java.util.Locale'](https://www.codacy.com/app/triplez/open-event-android/file/2408715845/issues/source?bid=3233850&fileBranchId=3234606#l11)

File: app/src/test/java/org/fossasia/openevent/utils/DateTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import java.util.TimeZone;
```
Comment (juslee) [inline app/src/test/java/org/fossasia/openevent/utils/DateTest.java]: Codacy Issue found: [Avoid unused imports such as 'java.util.TimeZone'](https://www.codacy.com/app/triplez/open-event-android/file/2408715845/issues/source?bid=3233850&fileBranchId=3234606#l12)

File: app/src/test/java/org/fossasia/openevent/utils/DateTest.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package org.fossasia.openevent.utils;
+
+import android.text.format.DateUtils;
```
Comment (juslee) [inline app/src/test/java/org/fossasia/openevent/utils/DateTest.java]: Codacy Issue found: [Avoid unused imports such as 'android.text.format.DateUtils'](https://www.codacy.com/app/triplez/open-event-android/file/2408715845/issues/source?bid=3233850&fileBranchId=3234606#l3)

**Comment by the-dagger** [comment]
@championswimmer Do we have to merge this?

**Pull request closed without merging**